doc/classes/CameraFeed.xml

```diff
@@ -34,6 +34,21 @@
 				Returns the position of camera on the device.
 			</description>
 		</method>
+		<method name="get_texture_tex_id">
+			<return type="int" />
+			<param index="0" name="feed_image_type" type="int" enum="CameraServer.FeedImage" />
+			<description>
+				Returns the texture backend ID (usable by some external libraries that need a handle to a texture to write data).
+			</description>
+		</method>
+		<method name="set_external">
+			<return type="void" />
+			<param index="0" name="width" type="int" />
+			<param index="1" name="height" type="int" />
+			<description>
+				Sets the feed as external feed : i.e provided by another library.
```

Comment:
```suggestion
				Sets the feed as external feed, i.e. provided by another library.
```

Reply:
Sorry but this would create a very weird sentence

Reply:
Using a colon like this is not correct English, do you mean to say `Sets the feed as external feed; i.e. provided by another library.`?

Reply:
I think the rewrite of this is ok for what its intended to be.